modules/ROOT/partials/configuration/tinycomments_reply.adoc

```diff
@@ -1,17 +1,15 @@
 [[tinycomments_reply]]
 == `+tinycomments_reply+`
 
-The Comments plugin uses the `+tinycomments_reply+` function to reply to a comment.
+The {pluginname} plugin uses the `+tinycomments_reply+` function to reply to a comment.
 
 The `+tinycomments_reply+` function saves the comment as a reply to an existing conversation and returns via the `+done+` callback once successful. Unrecoverable errors are communicated to {productname} by calling the `+fail+` callback instead.
 
 The `+tinycomments_reply+` function is given a request (req) object as the first parameter, which has these fields:
 
-`+conversationUid+`:: The uid of the conversation the reply is targeting.
-
-`+content+`:: The content of the comment to create.
-
-`+createdAt+`:: The date the comment was created.
+* `+conversationUid+`: The uid of the conversation the reply is targeting.
+* `+content+`: The content of the comment to create.
+* `+createdAt+`: The date the comment was created.
```

Comment:
Should we mention the format of `createdAt` ie: ISO-8601?

Reply:
```suggestion
* `+createdAt+`: The date the comment was created (ISO 8601 date string) format.
```